logic-app.tf

```diff
@@ -1,5 +1,5 @@
 resource "azurerm_resource_group_template_deployment" "api_connections_template_deployment" {
-  name                = "keyvault-api-connection-deploy"
+  name                = "${var.cluster_name}-keyvault-api-connection-deploy"
```

Comment:
why you don't have here prefix?

Reply:
i can add but its not really matter, more important to see which template resource assigned to cluster

Reply:
this is our convention for all resources names